subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -783,6 +787,70 @@ static int pgps_request_all(void)
 	return pgps_request(&request);
 }
 
+#if defined(CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_TRANSPORT_HTTP)
+/* handle incoming P-GPS response packets */
+int nrf_cloud_pgps_process(const char *buf, size_t buf_len)
+{
+	static char host[CONFIG_DOWNLOAD_CLIENT_MAX_HOSTNAME_SIZE];
+	static char path[CONFIG_DOWNLOAD_CLIENT_MAX_FILENAME_SIZE];
+	int err;
+	struct nrf_cloud_pgps_result pgps_dl = {
+		.host = host,
+		.host_sz = sizeof(host),
+		.path = path,
+		.path_sz = sizeof(path)
+	};
+
+	if (state == PGPS_NONE) {
+		LOG_ERR("P-GPS subsystem is not initialized.");
+		return -EINVAL;
+	}
+#if defined(CONFIG_NRF_CLOUD_MQTT)
+	LOG_HEXDUMP_DBG(buf, buf_len, "MQTT packet");
+#endif
+	if (!buf_len) {
+		LOG_ERR("Zero length packet received");
+		state = PGPS_NONE;
+		return -EINVAL;
+	}
+
+	if (ignore_packets) {
+		LOG_ERR("IGNORING PACKETS");
```

Comment:
```suggestion
		LOG_ERR("Ignoring packets");
```

are we able to tell the user why we are ignoring packets?

Reply:
Similarly, maybe rename the bool itself to "receiving_packets" and invert its meaning.

Also, since any thread can call nrf_cloud_pgps_process, I think ignore_packets should be an atomic type, or am I mistaken on that?